mmv1/third_party/terraform/website/docs/d/gke_hub_membership.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: "GKEHub"
+description: |-
+  Retrieves the details of a GKE Hub Membership.
+---
+
+# `google_gke_hub_membership`
+
+Retrieves the details of a specific GKE Hub Membership. Use this data source to retrieve the membership's configuration and state.
+
+## Example Usage
+
+```hcl
+data "google_gke_hub_membership" "example" {
+  project       = "my-project-id"
+  location      = "global"
+  name          = "my-membership-id" # GKE Cluster's name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The GKE Cluster name of the GKE Hub Membership id.
+
+* `location` - (Required) The location for the GKE Hub Membership.
+    Currently only `global` is supported.
+
+* `project` - (Optional) The ID of the project in which the resource belongs.
+    If it is not provided, the provider project is used.
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
```

Comment:
Usually we link to the resource documentation for exports on datasources so this information doesn't get out of sync as the resource evolves. Is there any reason we are not doing that here?

Reply:
No reason, I didn't get it.
I'll try to update it.

Reply:
Done